frontend/.stylelintrc.json

```diff
@@ -1,43 +1,60 @@
 {
   "plugins": [
+    "@stylistic/stylelint-plugin",
     "stylelint-order"
   ],
   "extends": [
-    "stylelint-config-recommended-scss",
+    "stylelint-config-standard",
+    "stylelint-config-standard-scss",
```

Comment:
Can I check why we are moving from the recommended config to the standard + standard scss config?

Reply:
`standard` was suggested by the [15.0.0 migration guide](https://stylelint.io/migration-guide/to-15) and `standard-scss` was suggested by the `recommended-scss` [github](https://github.com/stylelint-scss/stylelint-config-recommended-scss).

Do let me know if you want to keep it at the old configs!

Reply:
Got it, thanks! Let's go with these configs.